scripts/label_jupyter_and_rstudio_deployments.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+set -ex
+
+IDLE_LABEL=${1:-"mojanalytics.xyz/idleable"}
+
+for user in $(kubectl get ns | grep user- | cut -f 1 -d ' ' | sed 's/^user-//'); do
```

Comment:
Maybe I'm missing something but what about? 

```bash
for namespace in $(kubectl get ns | grep user- | cut -f1 -d" "); do
# ...
```
You're really interested in the namespaces right?

I mean maybe you used sed to be sure you're considering only namespaces starting with `user-`...potentially you could use `grep -e "^user-"` but maybe it's overkill?

Reply:
you're right, I just copied this from `analytics-platform-ops/scripts/upgrade_rstudio_pods.sh` without thinking too much. I'm not going to change it because it works and this is a throw away script :wastebasket:

Reply:
Happy with that ;)